### PR TITLE
Update quickstart with latest versions.

### PR DESCRIPTION
**Pull request by xrobau**
This simply adds 3.6 to the quickstart documentation.

### CODE REVIEW

File: source/documentation/quickstart/quickstart-guide.html.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The following requirements are typical for small- to medium-sized installations.
     -   [oVirt 3.3](http://resources.ovirt.org/pub/yum-repo/ovirt-release33.rpm)
     -   [oVirt 3.4](http://resources.ovirt.org/pub/yum-repo/ovirt-release34.rpm).
     -   [oVirt 3.5](http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm).
+    -   [oVirt 3.6](http://resources.ovirt.org/pub/yum-repo/ovirt-release36.rpm).
```
Comment (mykaul) [inline source/documentation/quickstart/quickstart-guide.html.md]: Why is this link different than the links below (the 'http://plain...' ones)?

**Comment by mykaul** [comment]
Personally, I'd remove all but 3.6 (possibly we can leave 3.5). There are also outdated references there (to Fedora 19, for example).

**Comment by xrobau** [comment]
I agree, I think it needs a good cleanup, but I don't think *I'm* qualified enough to do so, as I've been using it for less than 24 hours. Those were just some obvious mistakes that needed fixing.

**Comment by sandrobonazzola** [comment]
:+1: thanks for the patch. I agree with @mykaul that the whole document needs a massive rewrite, but I think that this patch is ok.

**Pull request merged**